launch.py

```diff
@@ -52,13 +63,15 @@ class LaunchTarget(Enum):
 
 
 def pull_image(image):
-    client = docker.client.from_env()
+    """
+    Try to pull the most up to date version of this image.
+    If internet connectivity is down, don't error.
+    """
     try:
-        client.images.pull(image)
-    except:
+        get_docker_client().images.pull(image)
+    except requests.exceptions.HTTPError:
```

Comment:
I've just confirmed the docker error thrown here is a derivative class of this error. Namely, it's a docker.errors.APIError, which is derived from requests.exceptions.HTTPError, so we're good here.